sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,299 @@
+------ DESKTOP SPONSORED TILES
+WITH newtab_unnested AS (
+  SELECT AS STRUCT
+    t.client_info.client_id,
+    date(t.submission_timestamp) AS submission_date,
+    t.sample_id,
+    s.name,
+    s.category,
+    s.extra
+  FROM
+    `mozdata.firefox_desktop.newtab` t
+  CROSS JOIN
+    UNNEST(t.events) s
+  WHERE
+    date(t.submission_timestamp) = @submission_date
+),
+desktop_events_1 AS (
+  -- desktop tiles clicks and impressions
+  SELECT
+    client_id,
+    submission_date,
+    COUNTIF(
+      name = "click"
+      AND category = "topsites"
+      AND mozfun.map.get_key(extra, "is_sponsored") = "true"
+    ) AS sponsored_tiles_click_count,
+    COUNTIF(
+      name = "impression"
+      AND category = "topsites"
+      AND mozfun.map.get_key(extra, "is_sponsored") = "true"
+    ) AS sponsored_tiles_impression_count,
+  FROM
+    newtab_unnested
+  WHERE
+    submission_date = @submission_date
+  GROUP BY
+    1,
+    2
+),
+desktop_events_2 AS (
+-- desktop Sponsored Tile Dismissals and Disables
+  SELECT
+    client_id,
+    DATE(submission_timestamp) AS submission_date,
+    COUNTIF(
+      event = 'BLOCK'
```

Comment:
Let's add this filter as well, otherwise, it will count the sponsored top stories dismissals as well.

```
AND source = 'TOP_SITES'
```

With the above filter, you can now drop `AND value LIKE '%card_type%'`.